irr/include/IOSOperator.h

```diff
@@ -31,10 +28,4 @@ class IOSOperator : public virtual IReferenceCounted
 	//! This is a no-op on some platforms.
 	//! \return Returns 0 if no string is in there, otherwise an utf-8 string.
 	virtual const c8 *getTextFromPrimarySelection() const = 0;
-
-	//! Get the total and available system RAM
-	/** \param totalBytes: will contain the total system memory in Kilobytes (1024 B)
-	\param availableBytes: will contain the available memory in Kilobytes (1024 B)
-	\return True if successful, false if not */
-	virtual bool getSystemMemory(u32 *totalBytes, u32 *availableBytes) const = 0;
```

Comment:
> I guess we should use `COSOperator::getSystemMemory` to constrain the thread choice.

https://github.com/luanti-org/luanti/pull/16634#issuecomment-3479509719 :eyes: 
I think this function can be helpful in the future, thus removing it now seems rash.

EDIT: However, the parameter names could be improved, perhaps `total_kB` or `totalKilobytes`, in that sense.